xonsh/jobs.py

```diff
@@ -443,3 +445,67 @@ def bg(args, stdin=None):
         _continue(curtask)
     else:
         return res
+
+
+def disown(args, stdin=None):
+    """
+    xonsh command: disown
+
+    Remove the specified jobs from the job table; the shell will no longer
+    report their status, and will not complain if you try to exit an
+    interactive shell with them running or stopped. If no job is specified,
+    disown the current job.
+
+    If the jobs are currently stopped and the $AUTO_CONTINUE option is not set
+    ($AUTO_CONTINUE = False), a warning is printed containing information about
+    how to make them continue after they have been disowned.
+
+    Specifying the -c or --cont option for this command is equivalent to
+    setting $AUTO_CONTINUE=True.
+    """
+
+    parser = argparse.ArgumentParser("disown", description=disown.__doc__)
+    parser.add_argument(
+        "job_ids",
+        type=int,
+        nargs="*",
+        help="Jobs to act on or none to use the current job",
+    )
+    parser.add_argument(
+        "-c",
+        "--cont",
```

Comment:
I'm all for ignoring outdated syntax and this seems like a good way to handle this.
My general take on `-` vs `--` args is that the `-` arg should be the abbreviation and `--` should be the verbose option.  Should this be `--continue` instead?